_config.yml

```diff
@@ -27,6 +25,14 @@ exclude:
 
 github: [metadata]
 
+plugins:
+  - jekyll-seo-tag
```

Comment:
@ConnorAhern Does this work with gh-pages? That could be the reason for failing. Read [here](https://stackoverflow.com/questions/53638446/jekyll-seo-tag-doesnt-work-within-github-pages)

Reply:
It does work with GH Pages. I have used it before.